src/components/SideNavMenu.tsx

```diff
@@ -0,0 +1,83 @@
+'use client';
+
+import { useState } from 'react';
+import { usePathname } from 'next/navigation';
+import Image from 'next/image';
+import Link from 'next/link';
+import ProfileImage from '@/components/ProfileImage'; // 기존 프로필 이미지 컴포넌트 사용
+
+// SVG 아이콘
+import MyInfo from '@/assets/icons/my-users.svg'; // 내정보
+import MyReservation from '@/assets/icons/my-reservations.svg'; // 예약내역
+import MyActivities from '@/assets/icons/my-activities.svg'; // 내 체험 관리
+import MyReservationStatus from '@/assets/icons/my-activities-dashboard.svg'; // 예약 현황
+import EditIcon from '@/assets/icons/pencil.svg';
+
+const NAV_ITEMS = [
+  { name: '내 정보', path: '/my-page', icon: MyInfo },
+  { name: '예약 내역', path: '/my-reservations', icon: MyReservation },
+  { name: '내 체험 관리', path: '/my-activities', icon: MyActivities },
+  { name: '예약 현황', path: '/reservation-status', icon: MyReservationStatus },
```

Comment:
아마 이 주소가 될 것 같습니다... 아마도...
```suggestion
  { name: '예약 현황', path: '/my-activities/[activityId]/reservation-dashboard', icon: MyReservationStatus },
```

Reply:
나중에 페이지에 얹고 나서 반응형 때문에 스타일 부분에서 수정사항이 있을 것 같습니다 현재로서는 제가 남긴 코드 리뷰만 반영해주시면 될 것 같습니다! 수고하셨습니다👍